index.html

```diff
@@ -5,7 +5,7 @@
     <title>ProjectHub</title>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.css">
     <link href='https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons' rel="stylesheet" type="text/css">
-    <link href="https://unpkg.com/vuetify@0.11.0/dist/vuetify.min.css" rel="stylesheet" type="text/css">
+    <link href="https://unpkg.com/vuetify@0.12.0/dist/vuetify.min.css" rel="stylesheet" type="text/css">
   </head>
```

Comment:
no spoko, tylko nie umiem jeszcze cssów dorzucić, ale Vuetify ma własny build we vue-cli i podpatrzę jak to robią :)